app/my_main.py

```diff
@@ -0,0 +1,50 @@
+
+from app.knights.knight import Knight
```

Comment:
Rewrite this file to main.py to see if it passes tests

Reply:
Thanks for the comment, but the test will not pass because of change the code